pkgs/development/python-modules/dbf/default.nix

```
@@ -19,10 +19,15 @@ buildPythonPackage rec {
     hash = "sha256-MFEi1U0RNvrfDtV4HpvPgKTCibAh76z7Gnmj32IubYw=";
   };
 
+  # Workaround for https://github.com/ethanfurman/dbf/issues/48
+  patches = lib.optional python.stdenv.isDarwin ./darwin.patch;
```

```suggestion
  patches = [ ./darwin.patch ];
```

But otoh we don't notice when it starts failing to apply.